cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -2,25 +2,44 @@
 from aws_cdk import aws_autoscaling as autoscaling
 from aws_cdk import aws_ec2 as ec2
 from aws_cdk import aws_elasticloadbalancingv2 as elbv2
+from aws_cdk import aws_iam as iam
 from constructs import Construct
 
+from pcluster.constants import EXTERNAL_SLURMDBD_ASG_SIZE
+
 
 class ExternalSlurmdbdStack(Stack):
-    def __init__(self, scope: Construct, id: str, **kwargs) -> None:
-        super().__init__(scope, id, **kwargs)
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+        self.stack = Stack(scope=scope, id=construct_id, **kwargs)
+
+        # define networking stuff
+        self.vpc_id = CfnParameter(
+            self, "VPC_id", type="String", description="The VPC to be used for the Slurmdbd stack."
```

Comment:
NitPick: Lets keep the Param name as CamelCase as done for all the others

Reply:
The commit is outdated